pkg/alertmanager/alertmanager_template.go

```diff
@@ -89,3 +92,35 @@ func withCustomFunctions(userID string) template.Option {
 		html.Funcs(funcs)
 	}
 }
+
+// loadTemplates produces a template.Template from several in-memory template files.
+// It is adapted from FromGlobs in github.com/prometheus/alertmanager/template/template.go
```

Comment:
You might want to add the lines and version to it in case the file gets changed or moved.
```suggestion
// It is adapted from FromGlobs in https://github.com/prometheus/alertmanager/blob/9de8ef36755298a68b6ab20244d4369d38bdea99/template/template.go#L67-L95
```